app/models/concerns/gias_helpers.rb

```diff
@@ -10,6 +10,8 @@ module GiasHelpers
     scope :in_england, -> { where("administrative_district_code ILIKE 'E%'") }
     scope :section_41, -> { where(section_41_approved: true) }
     scope :eligible, -> { currently_open.eligible_establishment_type.in_england.or(currently_open.in_england.section_41) }
+    scope :cip_only, -> { currently_open.where(school_type_code: GiasTypes::CIP_ONLY_TYPE_CODES) }
```

Comment:
This isn't quite right. If you're a CIP_ONLY type code, but have section 41 approval and are in England then you're fully eligible, not cip only

Reply:
Yeah this probably doesn't make sense as a scope in isolation as the query is going to add superfluous checks. Ill just add a test for the eligible_or_cip_only scope